nav2_tasks/include/nav2_tasks/compute_path_to_pose_action.hpp

```diff
@@ -37,6 +37,8 @@ class ComputePathToPoseAction
 
   void onInit() override
   {
+    BtActionNode<ComputePathToPoseCommand, ComputePathToPoseResult>::onInit();
```

Comment:
We could make it so the tasks don't override onInit. Instead, BtActionNode would provide an init function that gets overridden (maybe BtActionNodeInit). Then BtActionNode would handle onInit directly, do whatever it needs to do and then call the overridden BtActionNodeInit function.

That way the task implementer doesn't need to remember to call onInit everytime.